docs/getting-started/flashing-firmware/esp32/index.mdx

```diff
@@ -11,7 +11,7 @@ The recommended method for firmware flashing is the [Web-Based Installer.](https
 
 ## Flashing Method for ESP32 Devices
 
-1. The [Web-Based Installer](https://flasher.meshtastic.org) requires either Chrome or Edge browsers but is an excellent choice for quickly flashing devices. **This is the recommended method for firmware flashing, especially for those new to the project, due to its ease of use.**
+1. The [Web-Based Installer](/docs/getting-started/flashing-firmware/esp32/web-flasher.mdx) requires either Chrome or Edge browsers but is an excellent choice for quickly flashing devices. **This method is highly recommended for firmware flashing, especially for new users of the project, as it is easy to use.**
```

Comment:
Two things.

1. I'm surprised to se the web flasher instructions nested in the ESP32 pages (it'll work for both, though I know we want to push for drag and drop on nRF52)
2. Should we keep the link going to flasher.meshtastic.org and instead have a button added to that the flasher page back to the docs instructions?

Reply:
I must be confused on point 1, I thought it was capable of flashing the nrf52 devices as well!

I just took a look at the vercel preview you provided. It looks great. I have no issues with of the changes. I think you've added some additional clarity and the cta buttons are great to help the user click through the flow.